src/libcore/marker.rs

```diff
@@ -42,6 +42,104 @@ pub trait Sized {
 }
 
 /// Types that can be copied by simply copying bits (i.e. `memcpy`).
+///
+/// By default, variable bindings have 'move semantics.' In other
```

Comment:
Every value has move semantics by default, not just variable bindings, maybe "by default, types have 'move semantics."?